### PR TITLE
Support for recorded and paused time Fixes Part2

### PR DESCRIPTION
**Pull request by FJBDev**
Code fixes for [this issue](https://github.com/wolfgang-ch/mytourbook/issues/229#issuecomment-695181871) #229

General code improvements

### CODE REVIEW

**Comment by wolfgang-ch** [comment]
After the paused times are now working with my .fit data, it's a very cool feature :-)

Short pauses are now also visible in the timeline, without the displayed paused value, a pause could be hidden.

A problem is still to overlapped paused time which I already mentioned. When this should be solved, the tour marker labels should also be considered in the stacked labels which can be more complex because of the different tour marker label positions (vertical, horizontal)

mt-pauses

**Comment by wolfgang-ch** [comment]
I did some modifications in the pref page UI, see my commit comments in https://github.com/wolfgang-ch/mytourbook/commit/dc369ab7510f6e1dc40a9840e64938ab3094cea1

mt-pref-appearance

File: bundles/net.tourbook/src/net/tourbook/messages.properties [hunk 1]
```diff
@@ -3461,12 +3481,12 @@ merge_tour_source_graph_heartbeat_tooltip   = Merge heartbeat values from source
...
 
-pref_appearance_number_of_recent_tags         = Number of recent &tags:
+pref_appearance_number_of_recent_tags         = Number of recent &tags
 pref_appearance_number_of_recent_tags_tooltip = Number of recently used tags which are displayed in the context menu.\n\
                                                 \n\
                                                 0 will hide recently used tags.
-pref_appearance_pace_computation_option         = Pace computation options:
-pref_appearance_pace_computation_option_tooltip = This menu gives the ability to choose the time value to be used to compute and display pace values.
+pref_appearance_paceandspeed_computation_option         = Pace and Speed computation options
+pref_appearance_paceandspeed_computation_option_tooltip = This menu gives the ability to choose the time value to be used to compute and display pace and speed values.
```
Comment (wolfgang-ch) [inline bundles/net.tourbook/src/net/tourbook/messages.properties:3489]: The tooltip text `This menu gives...` references a menu but there is no a menu

Comment (wolfgang-ch) [reply bundles/net.tourbook/src/net/tourbook/messages.properties:3489]: `And` in a action/group label should not be written in Uppercase this was the reason why I renamed `Pace And Speed Display`

When I'm not sure to write in upper- or lower-case. I'm looking at the Eclipse UI.

mt-label-uppercase

Comment (FJBDev) [reply bundles/net.tourbook/src/net/tourbook/messages.properties:3489]: Good point, I will use this tip in the future

**Comment by FJBDev** [comment]
> After the paused times are now working with my .fit data, it's a very cool feature :-)
> 
> Short pauses are now also visible in the timeline, without the displayed paused value, a pause could be hidden.
> 
> A problem is still to overlapped paused time which I already mentioned. When this should be solved, the tour marker labels should also be considered in the stacked labels which can be more complex because of the different tour marker label positions (vertical, horizontal)
> 
> mt-pauses

Yes, now that the FIT issues are fixed, I will work on showing stacked labels using the complex code of `net.tourbook.ui.tourChart.ValueOverlapChecker`

File: bundles/net.tourbook/src/net/tourbook/messages.properties (hunk 1 above)
Comment (FJBDev) [reply bundles/net.tourbook/src/net/tourbook/messages.properties:3489]: And I have just removed the "menu" text.

**Pull request merged**